scripts/pipeline/ocp-cluster-e2e.sh

```diff
@@ -10,7 +10,14 @@ setup_env() {
     # Install kubectl and oc
     curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_CLIENT_VERSION}/openshift-client-linux.tar.gz | tar xvz
     sudo mv oc kubectl /usr/local/bin/
-
+    if [[ "$ARCHITECTURE" == "Z" ]]; then
+    {
+      echo "****** Installing kubectl-kuttl..."
+      curl -L -o kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.15.0/kubectl-kuttl_0.15.0_linux_x86_64
```

Comment:
Platform Z is typically s390x, so wondering if it should be `https://github.com/kudobuilder/kuttl/releases/download/v0.15.0/kubectl-kuttl_0.15.0_linux_s390x` ?

Reply:
Its installing onto the machine that kicks of the tests not the s390x cluster so the image is corrent